modules/ppcp-axo/resources/js/AxoManager.js

```diff
@@ -743,6 +785,34 @@ class AxoManager {
         return this.axoConfig?.widgets?.email === 'use_widget';
     }
 
+    delete_keys_with_empty_string = (obj) => {
```

Comment:
This should probably be camelCase

 ```suggestion
    deleteKeysWithEmptyString = (obj) => {
```